tests/conftest.py

```diff
@@ -70,10 +70,10 @@ def __init__(self, pixel_array, slice_position, row_cosine=None, column_cosine=N
 @pytest.fixture
 def axial_slices():
     return [
-        MockSlice(randi(*arbitrary_shape), 0),
-        MockSlice(randi(*arbitrary_shape), 1),
-        MockSlice(randi(*arbitrary_shape), 2),
-        MockSlice(randi(*arbitrary_shape), 3),
+        MockSlice(randi(*arbitrary_shape), 0, InstanceNumber=0),
+        MockSlice(randi(*arbitrary_shape), 1, InstanceNumber=1),
```

Comment:
The tests probably need at least one of
- a fixture where the InstanceNumber order did not match the slice_number order (which is used to calculate ImagePositionPatient). 
- a call of `combine_slices(sort_by_instance=True)`

Reply:
Good point, I will add these test cases!